admin_manual/configuration/server/security/password_policy.rst

```diff
@@ -0,0 +1,61 @@
+===============
+Password Policy
+===============
+
+.. _password_policy_label:
+
+.. figure:: ../../../images/configuration/server/security/password-policy-app.png
+   :alt: The Password Policy application
+
+From the 2.0.0 release of `the Password Policy app`_, ownCloud administrators (both enterprise **and** community edition) have the option of installing and enabling the application.
+The Password Policy application enables administrators to define password requirements for user passwords and public links.
+
+**User Passwords**
+
+- Specify valid password requirements.
+- Specify a password expiration period.
+- Specify expiration dates for public link shares.
+- Forced password change on first login.
+- Disallowing passwords that match a configurable number of previous passwords (defaults to the previous 3).
+- Users can be notified a configurable number of days before their password expires.
+- Users will be notified when their password has expired.
+
+**Public links**
+
+- Specify the number of days until link expires if a password is set.
+- Specify the number of days until link expires if a password is *not* set.
+
+.. note::
+   Active user sessions will **not** end when passwords expire.
+   However, a password change will be forced when the user session expires (e.g., on logout).
+   OAuth2 tokens for app or client authentication is not affected.
+
+.. note::
+   Installing and enabling the application also extends the occ command to support :ref:`the user\:expire-password command <user-expire-password_label>`.
+
+.. caution::
+   After enabling the "*days until user password expires*" policy setting in the web UI, administrators need to run the ``occ user:expire-password`` command to set an initial password change date for all existing users.
```

Comment:
I hope this is highlighted sufficiently so that administrators will not overlook it. A link to the section where you explain the usage of the occ command would be very useful, I think.

Reply:
I'll make it so.

Reply:
@pmaier1, below is what it will look like in the org version of the docs. The rounded orange box is only for emphasis in the screenshot.

![bildschirmfoto 2018-07-23 um 10 31 14](https://user-images.githubusercontent.com/196801/43066048-e7e776b0-8e63-11e8-884a-83cd65e7bcd8.png)

Reply:
Ah, so the link is in the box above, ok :+1: